metricflow-semantics/metricflow_semantics/model/semantics/metric_lookup.py

```diff
@@ -210,3 +210,18 @@ def get_min_queryable_time_granularity(self, metric_reference: MetricReference)
                     minimum_queryable_granularity = defined_time_granularity
 
         return minimum_queryable_granularity
+
+    def get_default_granularity_for_metrics(
```

Comment:
To confirm - what's the expected behavior when you have a derived metric that depends on two other metrics that have different default time granularities? Based on `SetDefaultGranularityRule`, it seems that the default granularity for parent metrics are not intended to be considered.

Reply:
That's correct. But now that you bring it up, I'm thinking I should confirm that behavior with product, so I'll post about it in #dev-metricflow and see what people think.
Thinking I can probably change this behavior on top of the stack if we decide to go that route.